### PR TITLE
Update pangoLEARN DM to accomodate new DB version

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
From version 3 onwards pangolin uses a new database schema, which I have labelled version 2.0. This update updates the pangoLEARN DM to mark all downloads from 2021-05-27 and later as version 2.0.

File: data_managers/data_manager_pangolearn/data_manager/pangolearn_dm.py [hunk 1]
```diff
@@ -139,11 +139,20 @@ def parse_date(d):
     ]
     for release in releases_to_download:
         tag = download_and_unpack(release["tarball_url"], output_directory)
+        release_date = parse_date(tag)
+        if args.pangolearn_format_version is not None:
+            version = args.pangolearn_format_version
+        else:
+            # 2021-05-27 was the first release of pangoLEARN for pangolin 3, which changed DB format
+            if release_date >= datetime.datetime(2021, 5, 27):
+                version = '2.0'
```
[the remark this answers is not in the document]
Comment (pvanheus) [reply data_managers/data_manager_pangolearn/data_manager/pangolearn_dm.py:148]: perhaps... but then keep the old stuff at version 1.0?

[the remark this answers is not in the document]
Comment (bgruening) [reply data_managers/data_manager_pangolearn/data_manager/pangolearn_dm.py:148]: Yes, we need to stick with this as its already deployed. There is no version 2.0 then. I think that is ok.

File: data_managers/data_manager_pangolearn/data_manager/pangolearn_dm.xml [hunk 2]
```diff
@@ -53,6 +53,18 @@
...
         </test>
+        <test>
+            <conditional name="release">
+                <param name="which" value="date_range" />
+                <param name="start_date" value="2021-05-27" />
+                <param name="end_date" value="2021-05-27" />
+            </conditional>
+            <output name="output_file">
+                <assert_contents>
+                    <has_text text='"format_version": "3.0"'/>
```
Comment (bgruening) [inline data_managers/data_manager_pangolearn/data_manager/pangolearn_dm.xml:64]: can you assert that to 1.0 in the above test, please as well?

**Pull request closed without merging**